config.toml.example

```diff
@@ -4,3 +4,26 @@
 
 # Speedrun related parameters
 konami_code = true
+
+[relics]
+"Solstice Diploma" = false
+"Amulet of Storytelling" = true
+"Guardian Aura" = true
+"Sequent Flare" = false
+"Truestrike Pendant" = false
+"Tome of Knowledge" = false
+"Falcon-eyed Parrot" = false
+"Salient Sails" = false
+"Gold Tooth" = false
+"Sixth Sense" = false
+"Adamant Shard" = false
+"Hidden Pockets" = false
+"Mithreel Rod" = false
+"Bearing Reel" = false
+"Stereofilament Line" = false
+"Tactician's Mettle" = false
+"Artful Gambit" = false
+"Double Edge" = false
+"Dubious Dare" = false
+"Solstice Doctorate" = false
+"Mark of the Speedrunner" = true
```

Comment:
You could technically bring these in as `RelicButton` and compare against those - its a bit uglier though, as the `array of tables` syntax is a bit weird.

https://toml.io/en/v1.0.0#array-of-tables

```toml
[[relics]]
name = "Solstice Diploma"
enabled = false

[[relics]]
name = "Amulet of Storytelling"
enabled = true
```

and the config becomes:

```rust
pub struct Config {
    pub konami_code: bool,
    pub relics: Vec<RelicButton>,
}
```

Reply:
I actually had it like that first, but then we added selected as well and it became weird.